app/scripts/modules/core/src/image/image.reader.ts

```diff
@@ -2,19 +2,19 @@ import { module } from 'angular';
 
 import { ProviderServiceDelegate, PROVIDER_SERVICE_DELEGATE } from 'core/cloudProvider/providerService.delegate';
 
-export interface IFindImageParams {
+export type IFindImageParams = {
```

Comment:
for some reason, `Record<>` types need don't work with `interface`s but do with `type`s
https://www.typescriptlang.org/play?#code/GYVwdgxgLglg9mABFApgZygCgPoA8BciASihHAE4AmAPBuTGAOYA0iYIAtgEYrkB8ASkQBvAL4AocQHopiAO4UA1mnFkwGRAwA2DFABUAngAcUhYYmBw4hdt16JRiALwiLVwgCYHAbnGoMmNq6hiYCvtKyCuTKfsYoiABiViHxLuaW1mycPOQOqggaAIZoKYRJcCnOrhmeef5YxSlhkjKIAJJglCi4iGgwjGCFUCDk8RwwaH1M4gyo5MCFEPHlbQtL1e5ZdrkSakVoq4umiVaH62lumQCMdegNB2sozUA